themes/beaver/assets/css/components/blocks/c-nav.css

```
@@ -0,0 +1,199 @@
+}
+
+/* Arrow indicator for dropdowns */
+.c-nav__arrow {
+  display: inline-block;
+  margin-left: 8px;
+  transform: rotate(0deg);
+  transition: transform 0.2s ease;
+  cursor: pointer;
+  /* Ensure arrow is clickable with adequate touch target */
+  padding: 8px;
+  margin: -8px;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-left



(lint/suspicious/noShorthandPropertyOverrides)